tests/commands/xml/test_enum.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
```

Comment:
We can delete this test as we test it implicit with the above one

Reply:
Done